src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>Components are defined as <code><s>a</s></code> Python classes which</blockquote><p> </p><blockquote>Since we know how to explicit<code><s>e</s></code>ly link the inputs to the output, this</blockquote><p> </p><blockquote>We must thus, import the component from OpenMDAO API ...</blockquote><p>-&gt; "Thus, we must import..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Changed in dfe233f7